.pre-commit-config.yaml

```diff
@@ -0,0 +1,37 @@
+exclude: (\.(js|svg)$)
+
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.6.0
+    hooks:
+      - id: check-builtin-literals
+      - id: check-case-conflict
+      - id: check-docstring-first
+      - id: check-executables-have-shebangs
+      - id: check-toml
+      - id: detect-private-key
+      - id: end-of-file-fixer
+      - id: trailing-whitespace
+  - repo: https://github.com/astral-sh/ruff-pre-commit
+    rev: v0.5.6
+    hooks:
+      - id: ruff
+        files: datashader/
+  - repo: https://github.com/hoxbro/clean_notebook
```

Comment:
Not sure if we should add this.

Reply:
Don't think so, at least for now. The examples notebooks are deliberately checked in with the output.

Reply:
I have restored the output of the notebooks